src/index.Rmd

```diff
@@ -23,23 +23,44 @@ survival analysis to illustrate the main features of the package.
 Enjoy reading!
 
 ## Quick start
-A simple stratified cross-validation of [linear discriminant analysis](&MASS::lda) with [%mlr].
-```{r}
+Applying a statistical algorithm with [%mlr] has a basic structure of 3 steps not depending on the specific model. 
```

Comment:
I would suggest sth. like:
"Here we show the [%mlr] workflow to train, make predictions, and evaluate a learner on a classification problem. It consists of 5 basic steps that are independent of the particular learning problem and method."

Please feel free to change and improve!